### ISSUE TITLE
Running failed

### ISSUE THREAD

**Issue by lilian132** [alabsi91/reanimated-color-picker#40]
### Description

Error: Exception in HostFunction: java.lang.UnsatisfiedLinkError: dlopen failed: cannot locate symbol "_ZN8facebook5react19ReadableNativeArrayC1EN5folly7dynamicE" referenced by...

![image](https://github.com/alabsi91/reanimated-color-picker/assets/4987042/37f7a0ff-bcd0-4071-b4c5-0ffdc16a5267)

### Steps to reproduce

1. npm i reanimated-color-picker
2. import ColorPicker, {
      Panel1,
      Swatches,
      Preview,
      OpacitySlider,
      HueSlider,
    } from 'reanimated-color-picker';

### Reanimated Color Picker version

2.3.2

### React Native version

0.72.3

### React Native Reanimated version

2.17.0

### React Native Gesture Handler version

2.12.1

### Platforms

Android

### JavaScript runtime

Hermes

### Workflow

React Native (without Expo)

### Architecture

Paper (Old Architecture)

### Build type

Debug mode

**Comment by alabsi91** [comment]
This issue is connected to the native side of React Native Reanimated. After a brief Google search, it appears that the Android device is using an older SDK version than the version targeted by the React Native app.

**Comment by lilian132** [comment]
> This issue is connected to the native side of React Native Reanimated. After a brief Google search, it appears that the Android device is using an older SDK version than the version targeted by the React Native app. You might want to consider using the Android emulator with the latest SDK version to see if that resolves the issue.

Thank you for your answer

I have successfully run it, it is indeed a problem with react-native-reanimated, but I am not sure about the details

In fact, the realistic react-native-reanimated version I initially installed was 3.4.2, but I kept reporting an issue that I couldn't ### solve([buildCMakeDebug](https://github.com/software-mansion/react-native-reanimated/issues/4241)). Later, I tried to lower the version but encountered this issue again. I guess it was all related to the environment. In the end, I couldn't help it. After uninstalling and reinstalling Android Studio, I reinstalled 3.4.2 and the problem disappeared.

The above text was generated by translation software and may have grammar issues.
